qs-api/qs-api-grpc/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>hu.icellmobilsoft.quarkus.sampler</groupId>
+		<artifactId>qs-api</artifactId>
+		<version>0.1.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>qs-api-grpc</artifactId>
+
+	<properties>
+		<maven.javadoc.skip>true</maven.javadoc.skip>
+		<version.io.grpc.version>1.60.0</version.io.grpc.version>
+		<version.protoc.version>3.21.7</version.protoc.version>
+		<version.os-maven-plugin.version>1.7.1</version.os-maven-plugin.version>
+		<version.hu.icellmobilsoft.coffee>2.1.0</version.hu.icellmobilsoft.coffee>
+	    <jakarta.annotation-api.version>1.3.5</jakarta.annotation-api.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>hu.icellmobilsoft.coffee</groupId>
+			<artifactId>coffee-dto-stub-gen</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>jakarta.annotation</groupId>
+			<artifactId>jakarta.annotation-api</artifactId>
+			<version>${jakarta.annotation-api.version}</version>
+		</dependency>
+	</dependencies>
+	<build>
+	  <resources>
+           <resource>
+                <directory>src/main/proto</directory>
+            </resource>
+        </resources>
+		<extensions>
+			<extension>
+				<groupId>kr.motd.maven</groupId>
+				<artifactId>os-maven-plugin</artifactId>
+				<version>${version.os-maven-plugin.version}</version>
+			</extension>
+		</extensions>
+		<plugins>
+			<!--          <plugin>-->
```

Comment:
Egy note-ot írsz miért van kikommentezve?

Reply:
ki is szedtem, köszi, bennemaradt, ez majd valahol akkor fog kellene ha az api-t szetszedjuk, stub generalas + proto reszre